src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -115,12 +116,27 @@ public string SetSubtitle(uint AppType, IPublicAPI api)
             {
                 return api.GetTranslation("powertoys_run_plugin_program_web_application");
             }
+            else if(AppType == (uint)ApplicationTypes.RUN_COMMAND)
+            {
+                return api.GetTranslation("powertoys_run_plugin_program_run_command");
+            }
             else
             {
                 return String.Empty;
             }
         }
 
+        public bool FilterPartialMatchForRunCommands(string query)

```

Comment:
`FilterPartialMatchForRunCommands` indicates that it would return true for matches and false otherwise. But it seems to be the opposite. Can you please update the function name to reflect it's behavior?

Reply:
I could remove the word Filter and rename it to `PartialMatchForRunCommands`. Let me know if you have any better suggestions.

Reply:
How about `IsNameNotEqualForRunCommands` or something similar to these lines?

Reply:
Inverted the logic of the function and renamed it to `QueryEqualsNameForRunCommands`.